src/simply-rets-api-helper.php

```diff
@@ -781,6 +781,38 @@ public static function srResidentialDetailsGenerator( $listing ) {
             $listing_ownership, "Ownership"
         );
 
+        // Compliance/compensation data
+        $complianceData = $listing->compliance;
+        $compensation = "";
+        foreach ($complianceData as $compKey => $compValue) {
+            // Normalize camelCase keys to words
+            $compKey = ucfirst(preg_replace('/(?<=\\w)(?=[A-Z])/', ' $1', $compKey));
+
+            // Add $/% signs where necessary
+            if(!strpos($compValue, "%") AND !strpos($compValue, "$")) {
```

Comment:
Is this a hard-coded check here? I'm a little weary of this. Also, in some cases the compensation type specified % or $. I might honestly just show the raw data for each field to start. then we can implement any hard-coding on top of that in later iterations.

Reply:
I removed all this so it's just going to show the raw data for each field (except `null`) and not append any symbols